microshift_networking/microshift-configuring-routes.adoc

```diff
@@ -0,0 +1,52 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="microshift-configuring-routes"]
+= Configuring routes
+include::_attributes/attributes-microshift.adoc[]
+include::_attributes/common-attributes.adoc[]
+:context: microshift-configuring-routes
+
+toc::[]
+
+You can configure routes in {microshift-short} for networked clusters.
+//Q: assume this is not applicable to anything that is disconnected or offline? what would be a microshift use case for configuring routes?
+
+//OCP module, edit with care; Creating an insecure/http route
+include::modules/nw-creating-a-route.adoc[leveloffset=+1]
+
+//Q: HTTP Strict Transport Security routes use projects; can MicroShift users still use these configs without logging in to a project? https://docs.openshift.com/container-platform/4.15/networking/routes/route-configuration.html#nw-enabling-hsts_route-configuration
+
+//Q: The sticky state stuff didn't seem applicable; lmk if it should be here? https://docs.openshift.com/container-platform/4.15/networking/routes/route-configuration.html#nw-using-cookies-keep-route-statefulness_route-configuration
```

Comment:
This makes sense in applications where you have more than 1 pod behind the router serving traffic for a route. Even though this might not be the common case for MicroShift I think it should still be worth noting? The router has that feature and some applications may have some kind of HA for traffic serving pods. @DanielFroehlich wdyt?

Reply:
Added it so we can read and review.